spec/unit/compressors/dmg_spec.rb

```diff
@@ -114,7 +114,7 @@ module Omnibus
               --detach-retries=5 \\
               --settings="#{subject.resource_path('settings.py')}" \\
               -Dbackground="#{subject.resource_path('background.png')}" \\
-              -Dpkg="#{package_dir}/project-1.2.3-2.pkg" \\
+              -Dpkg="#{package_dir}/project-1.2.3-2.x86_64.pkg" \\
```

Comment:
It seems curious that this is explict, and not `#{project.build_version}`, or some similar substitution. But, I'm an Omnibus noob, so what do I know.

Reply:
Well, in this spec (ruby unit test), these are part of the test setup:
https://github.com/DataDog/omnibus-ruby/blob/datadog-5.5.0/spec/unit/compressors/dmg_spec.rb#L11-L12

Looks fine to me.

Reply:
Oh.... I got it.  `/unit/` means test in this context. I was looking for "test" in the name - like everyone else in the world. I won't mourn the loss of Ruby when omnibus is gone.